metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -239,6 +252,20 @@ public TableDto deleteAndReturn(final QualifiedName name, final boolean isMView)
         return tableDto;
     }
 
+    private boolean hasTags(@Nullable final TableDto tableDto, final Set<String> hasTags) {
+        if (!hasTags.isEmpty() && tableDto != null) {
+            final Set<String> tags = getTableTags(tableDto.getDefinitionMetadata());
+            if (!tags.isEmpty()) {
+                for (String t: hasTags) {
```

Comment:
why not use the intersect()(or retainAll?) equivalent instead

Reply:
Used this since it iterates and exits out even if one matches. Using intersect or retailAll is much cleaner than what it is now.